### DISCUSSION TITLE
Local storage path persistence between k3s upgrades

### DISCUSSION THREAD

**Discussion by gtrubach** [k3s-io/k3s#3996] [Q&A] [answered]
Hi,

Is it possible to persist local storage path on node between upgrades? I manually modify local-path-config ConfigMap, but it resets after each k3s upgrade.

There is a CLI flag --defaul-local-storage-path but it's for default path only. Is there a way to specify node path via CLI?

**Comment by brandond** [comment, marked as answer]
Are you trying to persist a change to customize the path on specific nodes in your cluster?

**Reply by gtrubach** [reply]
Yes, exactly.

**Reply by brandond** [reply]
In that case you'd need to copy the modified local-storage manifest to a unique filename, make your changes, and then restart k3s with --disable=local-storage. K3s overwrites all the manifests for packaged components on start.

**Reply by gtrubach** [reply]
Got it, thanks!
